server/routes/threads.js

```diff
@@ -2,92 +2,50 @@ const express = require("express");
 const threadRouter = express.Router();
 const database = require("../database");
 
-//ALL ENDPOINTS HERE PRECEDED BY /THREADS e.g. /THREADS/THREAD1:
-
-//displaying all threads in forum main page: i.e. starting conversation post for each
 threadRouter.get("/", async (req, res) => {
   let sql = "SELECT * FROM threads";
 
   try {
     const [results] = await database.query(sql);
     res.status(200).json(results);
   } catch (error) {
-    res
```

Comment:
you need to add all this code back please. There's a lot of queries we use in there